exercises/ansible_collections/1-create-collections/README.md

```diff
@@ -0,0 +1,612 @@
+ok: [localhost]
+
+TASK [redhat.workshop_demo_collection.demo_image_builder : Generate greeting and store result] ********************************************************************************************************************
+ok: [localhost]
+
+TASK [redhat.workshop_demo_collection.demo_image_builder : Create build directory] ********************************************************************************************************************************
+ok: [localhost]
+
+TASK [redhat.workshop_demo_collection.demo_image_builder : Copy Dockerfile] ***************************************************************************************************************************************
+ok: [localhost]
+
+TASK [redhat.workshop_demo_collection.demo_image_builder : Copy custom index.html] ********************************************************************************************************************************
+changed: [localhost]
+
+TASK [redhat.workshop_demo_collection.demo_image_builder : Build and Push OCI image] ***************************************************************************************************************************************
+changed: [localhost]
+
+PLAY RECAP ********************************************************************************************************************************************************************************************************
+localhost                  : ok=7    changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+### Running the local container
+
+Locally test the built container image with podman to demonstrate the expected behavior
+of the collection's module and role.
+
+```bash
+podman run -d --rm -d -p 8080:80 localhost/demo-nginx
+```
+
+Test the nginx web server and inspect the body section to find the generated string in the h1 section.
+
+```bash
+$ curl localhost:8080
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Ansible Collections Workshop</title>
+  <meta name="description" content="Demo Nginx">
+  <meta name="author" content="gsalinet@redhat.com">
+
+  <link rel="stylesheet" href="css/styles.css?v=1.0">
+
+</head>
+
+<body>
+  <h1>
+    Hei Heisenberg!
+  </h1>
+</body>
+</html>
+```
+
+# Takeaways
+
+- Collections can be installed from Galaxy of from Red Hat Automation Hub. Default collections search paths or custom paths can be used.
+
+- Collections can be created using the `ansible-galaxy collection init` command. Users can develop collections contents accordingly to their needs and business logic.
+
+- Collections plugins can be either any kind of Ansible plugins or modules. Modules are often developed inside collection to create an autonomous lifecycle from the main Ansible upstream.
+
+- Collection roles can use local collections plugins and modules.
```

Comment:
Each exercise should link back to the main README and also to the next exercise so that people can actually navigate.

Reply:
I'm adding them